examples/esp32/esp32-server.rs

```diff
@@ -89,8 +89,15 @@ mod esp32 {
         #[allow(clippy::redundant_clone)]
         #[cfg(not(feature = "qemu"))]
         let (ip, _wifi) = {
-            let wifi = start_wifi(periph.modem, sys_loop_stack).unwrap();
-            (wifi.wifi().sta_netif().get_ip_info().unwrap().ip, wifi)
+            let wifi = start_wifi(periph.modem, sys_loop_stack).expect("failed to start wifi");
```

Comment:
since `start_wifi` returns an EspError instance can we print the error in the panic statement. This means calling `panic` instead of `expect` but I think it's worth the visibility (same comment applies to the corresponding change in the template)

Reply:
resolved offline, change unnecessary